.ci/logstash-run.sh

```diff
@@ -3,21 +3,20 @@ set -ex
 
 export PATH=$BUILD_DIR/gradle/bin:$PATH
 
+if [[ "$SECURE_INTEGRATION" == "true" ]]; then
+  ES_URL="https://elasticsearch:9200 -k"
+else
+  ES_URL="http://elasticsearch:9200"
+fi
+
 wait_for_es() {
-  echo "Waiting for elasticsearch to respond..."
-  es_url="http://elasticsearch:9200"
-  if [[ "$SECURE_INTEGRATION" == "true" ]]; then
-    es_url="https://elasticsearch:9200 -k"
-  fi
   count=120
-  while ! curl --silent $es_url && [[ $count -ne 0 ]]; do
+  while ! curl -s $ES_URL >/dev/null && [[ $count -ne 0 ]]; do
     count=$(( $count - 1 ))
-    [[ $count -eq 0 ]] && return 1
+    [[ $count -eq 0 ]] && exit 1
     sleep 1
   done
-  echo "Elasticsearch is Up !"
-
-  return 0
+  echo $(curl -s $ES_URL | python2 -c "import sys, json; print json.load(sys.stdin)['version']['number']")
```

Comment:
```suggestion
  echo $(curl -s $ES_URL | python -c "import sys, json; print(json.load(sys.stdin)['version']['number'])")
```
... should be future proof against Python 2/3